### ISSUE TITLE
HitBTCv2 fetch_order fails

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#668]
You should use https://api.hitbtc.com/?shell#trades-by-order for fetching HitBTC orders, and using parse_trades on them. 

Current code fails since https://api.hitbtc.com/?shell#orders-history doesn't return price for single order for some reason.

This is API response for single order:
```
[{'id': <redacted>, 'clientOrderId': '<redacted>', 'symbol': 'EOSBTC', 'side': 'sell', 'status': 'filled', 'type': 'limit', 'timeInForce': 'GTC', 'quantity': '75.000', 'cumQuantity': '75.000', 'createdAt': '<redacted>', 'updatedAt': '<redacted>'}]
```

**Comment by kroitor** [comment]
@nmiculinic , 

> Current code fails

What do you mean "fails" ? Also, there's a difference between a trade (and parse_trades) and a closed order (and the corresponding parse_order). Do not confuse the two, a closed order can be filled with multiple filling trades.

**Comment by nmiculinic** [comment]
By fail I mean when falling fetch_order it doesn't work; it throws exception:
```
TypeError: unsupported operand type(s) for *: 'float' and 'NoneType'
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "fino/tools/__main__.py", line 25, in <module>
    loop.run_until_complete(method())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/home/lpp/Desktop/fino/fino/tools/impl/order.py", line 48, in run
    info = await ex.fetch_order(id=self.cfg['order_id'])
  File "/home/lpp/Desktop/fino/fino/exchanges/_exchanges.py", line 78, in wrapper
    sol = await f(self, *args, **kwds)
  File "/home/lpp/Desktop/fino/fino/exchanges/_exchanges.py", line 1232, in fetch_order
    return await self.exchange.fetch_order(id=id)
  File "/home/lpp/Desktop/fino/.venv/lib/python3.6/site-packages/ccxt/async/hitbtc2.py", line 407, in fetch_order
    return self.parse_order(response[0])
  File "/home/lpp/Desktop/fino/.venv/lib/python3.6/site-packages/ccxt/async/hitbtc2.py", line 379, in parse_order
    cost = filled * price
TypeError: unsupported operand type(s) for *: 'float' and 'NoneType'
```

**Comment by kroitor** [comment]
I've added a temporary workaround for that particular issue, should be available in the upcoming version 1.10.204. Thx!
